account_payment_order/models/account_move.py

```diff
@@ -151,6 +151,13 @@ def create_account_payment_line(self):
                         "order": payment_lines.order_id.mapped("name"),
                     }
                 )
+
+            # Check that the bank allows out payments
+            for line in applicable_lines.filtered(
+                lambda l: l.account_id.account_type == "liability_payable"
+            ):
+                line._check_bank_allows_out_payments()
```

Comment:
As you already loop in your function, you can skip one level:
`applicable_lines.filtered(lambda l: l.account_id.account_type == "liability_payable")._check_bank_allows_out_payments()`